code/game/objects/items/toys.dm

```diff
@@ -123,6 +119,52 @@
 	user.visible_message("<span class='notice'>[user] plays with [src].</span>", "<span class='notice'>You [playverb].</span>")
 	lastused = world.time
 
+/*	Coloured balloons, sprite taken from Polaris Station */
+
+/obj/item/toy/colorballoon
+	name = "balloon"
+	desc = "Colourful fun for all ages! Alien plastic alloys have made it exceptionally hard to pop!"
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "colorballoon" /* generic version of this balloon. you can edit its color var to a hex code including # to give it colour, but i wasn't able to get the in hand version to work with changing color vars for some reason */
+	item_state = "colorballoon"
+	w_class = WEIGHT_CLASS_NORMAL
+	var/lastused = null
+
+/obj/item/toy/colorballoon/attack_self(mob/user)
+	if(world.time - lastused < CLICK_CD_MELEE)
+		return
+	var/playverb = pick("bat [src]", "tug on [src]'s string", "play with [src]")
+	user.visible_message("<span class='notice'>[user] plays with [src].</span>", "<span class='notice'>You [playverb].</span>")
+	lastused = world.time
```

Comment:
To avoid copy and paste here, it would be nice if you could make an `/obj/item/toy/balloon` subtype so that current and future balloons can share this proc.

Reply:
Good point, I'll do so!